tests/multigrid/dof_06.output

```diff
@@ -0,0 +1,88 @@
+--- DIM 2 OK ---
```

Comment:
I believe that this is not the output that is generated by running `ctest`.
For generating the conforming output, run 
```
$ make setup_tests_multigrid
```
and
```
$ ctest -R "multigrid/dof_06"
```
in your build directory and copy the output via
```
cp tests/multigrid/dof_06.debug/output ${SOURCE_DIR}/tests/multigrid/dof_06.output
```
to your source directory, i.e. this file.

Reply:
That's exactly what I did, I just ran the test using ctest and it passed.

Reply:
Will address all points that I did not further comment on directly, the two I commented on need some discussion.

Reply:
> That's exactly what I did, I just ran the test using ctest and it passed.

Great!